code/go/0chain.net/smartcontract/stakepool/stakepool.go

```diff
@@ -200,6 +200,24 @@ func (sp *StakePool) OrderedPoolIds() []string {
 	return ids
 }
 
+// GetOrderedPools returns a slice of ordered pools
+func (sp *StakePool) GetOrderedPools() []*DelegatePool {
```

Comment:
should be fixed in staging by @din-mukhammed

Reply:
Yes, should have other methods to get the order values, I'll change it later.

Reply:
Hmm, the staging added `OrderedPoolIds()`, but we will need extra map calling to get the pool, which I think in most cases is unnecessary. So should be fine to keep this `GetOrderedPools()` so that we can use the ordered pools directly.